src/components/dataVisualizations/columnHelpers/commanderOverviewColumnHelper.tsx

```diff
@@ -24,22 +24,22 @@ export const commanderOverviewColumns: ColumnDef<Commander, any>[] = [
         },
         header: () => <span>Name</span>
     }),
-    columnHelper.accessor((row) => row.matches.length, {
+    columnHelper.accessor((row) => row.validMatchesCount, {
         id: "gameCount",
-        cell: (info) => info.row.original.matches.length,
+        cell: (info) => info.row.original.validMatchesCount,
         header: () => <span>Game Count</span>
     }),
     columnHelper.accessor((row) => row.wins, {
         id: "wins",
         cell: (info) => info.row.original.wins,
         header: () => <span>Wins</span>
     }),
-    columnHelper.accessor((row) => (row.matches.length > 0 ? Math.round((row.wins / row.matches.length) * 100) : 0), {
+    columnHelper.accessor((row) => (row.validMatchesCount > 0 ? Math.round((row.wins / row.validMatchesCount) * 100) : 0), {
         id: "winrate",
         cell: (info) =>
-            info.row.original.matches.length > 0
-                ? `${Math.round((info.row.original.wins / info.row.original.matches.length) * 100)}%`
-                : `0%`,
+            info.row.original.validMatchesCount > 0
+                ? `${Math.round((info.row.original.wins / info.row.original.validMatchesCount) * 100)}%`
```

Comment:
since you are here already, can we update this to use the getWinRatePercentage method?

Reply:
This will have to wait until a bit later. That's a negative if we want to push this out soon. (But yes, I think updating that is a good idea.)

Reply:
No worries, we can do it later.

Reply:
Done.

Reply:
Added some documentation as well.